bin/cozy-light

```diff
@@ -88,16 +94,17 @@ if (!process.argv.slice(2).length) {
 
 // Manage errors
 
-process.on('uncaughtException', function (err) {
+process.on('uncaughtException', function(err){
   if (err) {
-    logger.warn('An exception is uncaught');
-    logger.raw(err);
-    logger.raw(err.stack);
-    actions.exit();
+    cozyLight.logger.warn('An exception is uncaught');
+    cozyLight.logger.raw(err);
+    cozyLight.logger.raw(err.stack);
   }
+  cozyLight.stop(cozyLight.exit);
```

Comment:
Yes, see #62. We shall catch the exception within actions.js i think. When it s caught in uncaughtExeption, i believe the stack is already dead and nothing can be done. Also needs to see how we can identify the problematic app.
Many things, i leave it for another PR.